R/estimateDominance.R

```diff
@@ -38,6 +38,57 @@
+#' abundances of the two most dominant species and $N_{tot}$ is the sum of absolute
+#' abundances of all species.) Index gives values in interval [0,1],
+#' where bigger value represent greater dominance.}
+#' \item{'absolute' }{Absolute index equals to the absolute abundance of the most
+#' dominant species of the sample. Index gives positive integer values.}
+#' \item{'relative' }{Relative index equals to the relative abundance of the most
+#' dominant species of the sample. ($\frac{N_1}{N_{tot}}$, where $N_1$ is the absolute abundance of
+#' the most dominant species and $N_{tot}$ is the sum of absolute abundances of all species.)
+#' Index gives values in interval [0,1], where bigger value represent greater dominance.}
+#' \item{'simpson' }{Simpson's index, or Simpson's dominance index, is calculated
+#' by raising all relative abundances of species to the power of 2, and then summing them together.
+#' ($\sum(p^2)$, where $p$ is relative abundances.) Index gives values in interval [0,1].
+#' Value equals the probability that two randomly chosen individuals belongs to the same species.
+#' The higher the probability, the greater the dominance is. (Thukral et al. 2019.)}
+#' \item{'core_abundance' }{Core abundance index is related to core species.
+#' Core species are species that are most abundant in all samples, i.e., in whole data set.
+#' Core species are defined as those species that have prevalence over 50\%.
+#' It means that in order to belong to core species, species must be prevalent in 50\% of samples.
+#' Core species are used to calculate the core abundance index.
+#' Core abundance index is sum of relative abundances of core species in the sample.
+#' ($\frac{N_{core}}{N_{tot}}$, where $N_{core}$ is the sum of absolute abundance of the core species and
+#' $N_{tot}$ is the sum of absolute abundances of all species.) Index gives values in interval [0,1],
+#' where bigger value represent greater dominance.}
+#' \item{'gini' }{Gini index is probably best-known from socio-economic contexts.
+#' In economics, it is used to measure, for example, how unevenly wealth
+#' is distributed among population. Here, Gini index is used similarly, but wealth
+#' is replaced with abundance. If there is small group of species that represent
+#' large portion of total abundance of microbes, the inequality is large and
+#' Gini index closer to 1. If all species has equally large abundances, the equality
+#' is perfect and Gini index equals 0. (Sitthiyot & Holasut 2020.)}
+#' }
+#'
+#' @references
+#' Maiti SK (2012) Ecorestoration of the coalmine degraded lands.
+#'
+#' Sitthiyot T & Holasut K (2020) A simple method for measuring inequality.
+#' Palgrave Commun 6(112) doi: 10.1057/s41599-020-0484-6.
+#'
+#' Thukral AK, Bhardwaj R, Kumar V & Sharma A (2019)
+#' New indices regarding the dominance and diversity of communities,
+#' derived from sample variance and standard deviation.
+#' Heliyon 5(10). doi: 10.1016/j.heliyon.2019.e02606.
+#'
```

Comment:
Also here, if any more original/classical references can be identified that would be preferable.

Reply:
Changed to 

Simpson EH (1949) Measurement of Diversity. Nature 163(688). doi: 10.1038/163688a0